### PR TITLE
delete

### PR DESCRIPTION
**Pull request by jeesupark96**
image

### CODE REVIEW

File: express-delete/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+};
+
+app.get('/api/grades', (req, res, next) => {
+  const newarray = [];
+  for (const prop in grades) {
+    const result = grades[prop];
+    newarray.push(result);
+  }
+  res.json(newarray);
+  next();
```
Comment (scbowler) [inline express-delete/index.js]: We only want to call `next` if there is a next thing to do. Once we send a response with `res.json` the request has completed and there is nothing else to do or that should be done. So we don't want to use `next` here.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just one small update needed. Let me know if you have any questions.

File: express-delete/index.js (hunk 1 above)
Comment (jeesupark96) [reply express-delete/index.js]: so in the case of calling next, that would be if we had multiple of the same app.methods?

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**